axelrod/tests/unit/test_resultset.py

```diff
@@ -394,3 +392,488 @@ class TestDecorator(unittest.TestCase):
     def test_update_progress_bar(self):
         method = lambda x: None
         self.assertEqual(axelrod.result_set.update_progress_bar(method)(1), None)
+
+
+class TestResultSet_SpatialStructure(TestResultSet):
```

Comment:
Minor comment: `TestResultSet_SpatialStructure` PEP8 says no underscores for class names (just CamelCase).